src/server/chromium/crNetworkManager.ts

```diff
@@ -620,9 +629,7 @@ class ResponseExtraInfoTracker {
   }
 
   responseReceivedExtraInfo(event: Protocol.Network.responseReceivedExtraInfoPayload) {
-    const info = this._requests.get(event.requestId);
-    if (!info)
-      return;
+    const info = this._getOrCreateEntry(event.requestId);
```

Comment:
What if we called `this._stopTracking()` for this info already, and then received extra info for the 10th redirect? It seems like we'll create a new info that will never clean up?

Reply:
That is the case. It is unlikely though and I don't know how to mitigate it...